src/commands/run.js

```diff
@@ -43,12 +45,23 @@ export async function run({ taskName, options }) {
   if (logger instanceof InteractiveLogger) logger.clearTasks()
 
   const failedTasks = tasks.allFailedTasks()
-
   if (failedTasks.length > 0) {
     logger.fail(`Failed tasks: ${failedTasks.join(', ')}`)
   }
 
-  if (Object.entries(tasks.allTasks).every(([, task]) => task.status === 'success:lazy')) {
-    logger.log('\n' + rainbow('>>> MAXIMUM LAZY'))
-  }
+  const stats = tasks.getTaskStats()
+  const successOutput = `${kleur.green(stats.successful.toString() + ' successful')}, ${
+    stats.allTasks
+  } total`
+
+  const cachedOutput =
+    stats.allTasks === stats['success:lazy']
+      ? rainbow('>>> MAXIMUM LAZY')
+      : `${stats['success:lazy']} cached, ${stats.allTasks} total`
+
+  const output = dedent`
+    
+          Tasks:     ${successOutput}
+         Cached:     ${cachedOutput}`
```

Comment:
Using `Cached:` prefix even for the `>>> MAXIMUM LAZY`. I tried a few positions without it and the alignment just looked wrong to me, no matter what I tried. Feel free to drop it if you'd like though.